frontend/kesaseteli/employer/src/components/application/steps/step2/error-notification/EmploymentsErrorNotification.sc.ts

```diff
@@ -0,0 +1,9 @@
+import { Notification } from 'hds-react';
+import { $GridCell } from 'shared/components/forms/section/FormSection.sc';
+import styled from 'styled-components';
+
+export const $NotificationDescription = styled($GridCell)``;
```

Comment:
Why is this needed?

Reply:
Perhaps there was a thought of using more descriptive name for the component than GridCell but I can remove this.

Reply:
Removed